views/components/Sidebar/SidebarFooter.jsx

```diff
@@ -10,51 +10,62 @@ import {
 } from '@mui/material'
 import { Link } from 'react-router-dom'
 import { routes } from '../../routes/routes'
-import { FontSize14InRem } from '../../../constants'
+import { fontSize } from '../../../constants'
 import './SidebarFooter.css'
 
 const SidebarFooter = ({ user, onLogout }) => {
   return (
-    <Card sx={{ border: 0, boxShadow: 0, color: 'text.primary' }}>
+    <Card sx={{ border: 0, boxShadow: 0, marginTop: 'auto' }}>
```

Comment:
The text color was removed on accident

Reply:
we don't need to specify the text color here.

Reply:
The default mui black is a different shade than the default text color we are using in the app.

<img width="803" alt="Screenshot 2023-11-08 at 3 15 26 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/235d29e0-eec5-4084-94e3-4d22f11c1fc1">

Reply:
right but isn't MUI using the primary text color by default and we're configuring that in our theme?